Dockerfile.lab

```diff
@@ -0,0 +1,18 @@
+FROM node:22 AS builder
+
+ARG NEXT_PUBLIC_COMMIT_HASH
+ENV NEXT_TELEMETRY_DISABLED=1
+ENV PORT=8100
+WORKDIR /lab
+RUN git clone https://github.com/stellar/laboratory /lab
+RUN git fetch origin ${NEXT_PUBLIC_COMMIT_HASH}
+RUN git checkout ${NEXT_PUBLIC_COMMIT_HASH}
+RUN rm -rf .git
+RUN yarn install
```

Comment:
`yarn install` is not guaranteed to be reproducible. We need to add the `--frozen-lockfile` option to ensure it is. Can we do that here to make sure that we never build dependencies into the quickstart image that haven't been tested and run through the normal test processes of lab?

Reply:
- https://github.com/stellar/quickstart/issues/698